### PR TITLE
Add Inferentia test api

### PR DESCRIPTION
**Pull request by deliahu**

### CODE REVIEW

File: test/apis/realtime/image-classifier-resnet50/inferentia.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Inferentia
+
+The Inferentia examples were inspired by [this tutorial](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-deploy/tutorials/k8s_rn50_demo.html).
```
Comment (RobertLucian) [inline test/apis/realtime/image-classifier-resnet50/inferentia.md:3]: With respect to the entire `md` file: maybe we can add a comment explaining that this should naturally be hit from outside the cluster, not by `exec`ing into the pod and running these. What do you think?

File: test/apis/realtime/image-classifier-resnet50/cortex_inf.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+- name: image-classifier-resnet50
+  kind: RealtimeAPI
+  pod:
+    port: 9000
+    max_concurrency: 8
+    containers:
+    - name: api
+      image: quay.io/cortexlabs-test/neuron-tf-serving:latest
```
Comment (RobertLucian) [inline test/apis/realtime/image-classifier-resnet50/cortex_inf.yaml:8]: What do you think of adding this to the E2E test suite?

File: test/apis/realtime/image-classifier-resnet50/cortex_inf.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      command: ["/usr/local/bin/entrypoint.sh"]
+      args:
+        - --port=8500
+        - --rest_api_port=9000
```
[the remark this answers is not in the document]
Comment (deliahu) [reply test/apis/realtime/image-classifier-resnet50/cortex_inf.yaml:12]: This should work from outside

File: test/apis/realtime/image-classifier-resnet50/cortex_inf.yaml (hunk 2 above)
Comment (deliahu) [reply test/apis/realtime/image-classifier-resnet50/cortex_inf.yaml:8]: Yes, I will create a separate ticket for this

**Review by RobertLucian — APPROVED** [review]

**Pull request merged**